### ISSUE TITLE
Option to disable crisp mode of Canvas (kjua) library

### ISSUE THREAD

**Issue by rugk** [rugk/offline-qr-code#91]
One problem the crisp setting is causing is https://github.com/rugk/offline-qr-code/issues/90.

So we could make it optional, so one can use the Canvas without the crisp setting.

BTW the SVG version has no pixel-perfect (crisp) rendering mode.

This requires:
* a new QR code size option
* this option should be dynamically disabled (and unchecked!) when the SVG version is chosen
* when the Canvas is choosen, it should be enabled (and if there is a setting, the old state of the setting should be restored)
* by default the setting should be enabled
* adjustments in the popup of this setting: 
   https://github.com/rugk/offline-qr-code/blob/137a2a3533e5786b47e1b4f6a0e070d8fd87119a/src/popup/qrcode.js#L172

**Comment by mitcheesta** [comment]
I'd like to give this a try. I'm a first-time contributor so please let me know if there's anything I'm doing wrong.

**Comment by rugk** [comment]
Yeah, thanks. Sure. Let me know if you have any questions. :smiley:

**Comment by rugk** [comment]
@mitcheesta Are you still interested in it? If not, that's no problem, just add a notice, so I know what the current status of this issue is.

**Comment by heanzyzabala** [comment]
Hi, I would like to give this a try.

**Comment by rugk** [comment]
Thanks, sure, feel free to take this on. :smiley:

**Comment by heanzyzabala** [comment]
@rugk Thanks!

**Comment by rugk** [comment]
@heanzyzabala If you need any pointers/help or have questions, feel free to ask.
